src/test/java/com/google/sps/ContextServletTest.java

```diff
@@ -43,15 +48,34 @@
 /** This is a JUnit test for sentiment mockedAnalysis */
 @RunWith(JUnit4.class)
 public class ContextServletTest {
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
   private static final String URL_PARAMETER = "url";
   private static final String TEST_URL = "WkZ5e94QnWk";
   private static final String TEST_TITLE = "Test Title";
   private static final String TEST_CHANNEL = "Test Channel";
+  private static final String REQUEST_INFO = "snippet,statistics";
   private static final int NUM_LIKES_VAL = 10;
   private static final int NUM_DISLIKES_VAL = 1;
 
   @Test
-  public void testContextGet() throws IOException, GeneralSecurityException {
+  public void testYoutubeAPIConstructed() throws IOException, GeneralSecurityException {
+    // Test the constructeExecuteYoutubeReuest method for returning a youtube request
+    YouTube youtubeService =
+        new YouTube(GoogleNetHttpTransport.newTrustedTransport(), JSON_FACTORY, null);
+    YouTube mockedYoutubeService = spy(youtubeService);
+    YouTube.Videos mockedVidoes = mock(YouTube.Videos.class);
+    YouTube.Videos.List mockedVideoList = mock(YouTube.Videos.List.class, RETURNS_DEEP_STUBS);
+    ContextServlet contestServlet = new ContextServlet(mockedYoutubeService);
+    when(mockedVidoes.list(REQUEST_INFO)).thenReturn(mockedVideoList);
+    when(mockedYoutubeService.videos()).thenReturn(mockedVidoes);
+    when(mockedVideoList.execute()).thenReturn(new VideoListResponse());
+    VideoListResponse response = contestServlet.constructExecuteYouTubeRequest(TEST_URL);
+    Assert.assertEquals(response.getClass(), VideoListResponse.class);
+    Assert.assertEquals(response.getItems().size(), 0);
+  }
+
+  @Test
+  public void testContextGet() throws IOException, GeneralSecurityException, ServletException {
```

Comment:
Maybe this can be changed to reflect that it's primarily concerned with verifying that the responses get generated as expected?

Perhaps something like `testContextGet_generatesExpectedResponse`?